src/middleware/middleware.ts

```diff
@@ -4,11 +4,15 @@ import { getPayload } from "./get-payload";
 import { verifyAndReceive } from "./verify-and-receive";
 import { debug } from "debug";
```

Comment:
Keeping `debug` package for retrocompatibility and creating issue to remove it in a different breaking change